test/ruby_lsp_rails/server_test.rb

```diff
@@ -176,6 +176,28 @@ def resolve_route_info(requirements)
     assert_equal("Hello\n", stderr)
   end
 
+  test "checking for pending migrations" do
+    capture_subprocess_io do
+      system("bundle exec rails g migration CreateStudents name:string")
+    end
+
+    @server.execute("pending_migrations", {})
+    message = response.dig(:result, :pending_migrations)
```

Comment:
Since `pending_migrations` doesn't contain a list of pending migration names but just a message, I wonder if we can skip this namespace? I find it not too useful but a bit misleading instead.